doc/introduction/index.rst

```diff
@@ -162,6 +162,10 @@ The Boot process proceeds as follows:
    bootloader through dm-verity
 #. Virtual bootloader starts the User-side verified boot process
 
+.. note::
+   To avoid hardware resources conflict with ACRN hypervisor, UEFI
+   services shall not use IOMMU.
+
```

Comment:
could you also help to add timer service limitation?

Currently we only support UEFI timer with HPET MSI.

Reply:
New version has been pushed with timer service limitation.